release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -124,6 +124,13 @@ For more information, see xref:../installing/installing_azure/installing-azure-c
 
 This update introduces a new option for Ingress Controllers with the `hostnetwork` endpoint strategy. You can now host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports.
 
+[id=ocp-4-11-multiple-configuration-control-worker-nodes]
+==== Multi-node configuration for controller and worker nodes
+
+When you configure nodes, you can use a mechanism that applies a single configuration to multiple, bare metal installer-provisioned infrastructure (IPI) nodes in a cluster. Applying a single configuration that defines multiple nodes in your cluster reduces the risk of misconfiguration due to the single-provisioning process.
+
+The controller node and the worker nodes are configured using separate deployments.
```

Comment:
This isn't necessarily true. It's possible to configure all the nodes in a deployment in install-config. The only time a different process is needed is on scale out because the installer isn't run so you can't use install-config.

If we want to note the difference between day 1 and day 2 configuration, it's probably better to mention that this mechanism only applies to install-config and thus can only be used at initial deployment time.